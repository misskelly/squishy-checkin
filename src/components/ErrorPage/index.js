import React from 'react';

const ErrorPage = () => (
  <div>
      404 page
  </div>
);

export default ErrorPage;
