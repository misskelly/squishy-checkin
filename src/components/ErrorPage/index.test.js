import React from 'react';
import { shallow } from 'enzyme';
import ErrorPage from '.';

describe('ErrorPage', () => {
  let wrapper;
  beforeEach(() => {
    wrapper = shallow(<ErrorPage />);
  });

  it('should match the snapshot', () => {
    expect(wrapper).toMatchSnapshot();
  });
});
