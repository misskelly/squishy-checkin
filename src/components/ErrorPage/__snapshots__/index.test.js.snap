// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ErrorPage should match the snapshot 1`] = `ShallowWrapper {}`;
